### PR TITLE
Adopt Gradle over Make

### PR DESCRIPTION
**Pull request by rmfitzpatrick**
**Description:**
Feature addition - Removes Make usage from project in favor of gradle.

**Existing Issue(s):**

https://github.com/open-telemetry/opentelemetry-java-contrib/issues/11

**Testing:**

Should be picked up by existing checks

**Documentation:**

Updated existing docs as necessary

### CODE REVIEW

**Review by carlosalberto — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -21,10 +21,19 @@ allprojects {
         jcenter()
         mavenCentral()
     }
+}
+
+task lint(dependsOn: ['spotlessApply'])
```
Comment (anuraaga) [inline build.gradle:26]: Is it just `lint` => `spotlessCheck`? I'm not sure I've seen lint refer to a fixing step before.

**Review by anuraaga — APPROVED** [review]
Thanks!

File: build.gradle (hunk 1 above)
Comment (rmfitzpatrick) [reply build.gradle]: Thanks, this is term misusage on my end.  I've associated linting w/ the fix features of eslint and flake8.  Corrected for `lint` to be the check and `format` for spotlessApply.

**Comment by rmfitzpatrick** [comment]
Bypassing EasyCLA, which appears to not be working across otel projects atm.

**Pull request merged**